### PR TITLE
[RHCLOUD-23628] update compliance templates

### PR DESCRIPTION
**Pull request by g-duval**
image

image

image

### CODE REVIEW

File: common/src/main/java/com/redhat/cloud/notifications/config/FeatureFlipper.java [hunk 1]
```diff
@@ -72,6 +72,9 @@ public class FeatureFlipper {
     @ConfigProperty(name = "notifications.use-policies-email-templates-v2.enabled", defaultValue = "false")
     boolean policiesEmailTemplatesV2Enabled;
 
+    @ConfigProperty(name = "compliance.use-policies-email-templates-v2.enabled", defaultValue = "false")
```
Comment (josejulio) [inline common/src/main/java/com/redhat/cloud/notifications/config/FeatureFlipper.java:75]: I suppose you mean:

```suggestion
    @ConfigProperty(name = "notifications.use-compliance-email-templates-v2.enabled", defaultValue = "false")
```

File: engine/src/main/java/com/redhat/cloud/notifications/templates/TemplateService.java [hunk 2]
```diff
@@ -25,6 +25,8 @@
 @ApplicationScoped
 public class TemplateService {
 
+    public static final String HCC_LOGO_TARGET = "Logo-Red_Hat-Hybrid_Cloud_Console-A-Reverse-RGB";
```
Comment (josejulio) [inline engine/src/main/java/com/redhat/cloud/notifications/templates/TemplateService.java:28]: Same as the other PR - Lets move this variable to test/**

**Review by josejulio — CHANGES_REQUESTED** [review]

File: common/src/main/java/com/redhat/cloud/notifications/config/FeatureFlipper.java (hunk 1 above)
Comment (g-duval) [reply common/src/main/java/com/redhat/cloud/notifications/config/FeatureFlipper.java]: oups :)

File: engine/src/main/java/com/redhat/cloud/notifications/templates/TemplateService.java (hunk 2 above)
Comment (g-duval) [reply engine/src/main/java/com/redhat/cloud/notifications/templates/TemplateService.java]: Yep, I moved it

**Review by josejulio — APPROVED** [review]
lgtm

**Pull request merged**